tuxemon/core/states/combat/combat.py

```diff
@@ -96,6 +97,8 @@ def load_technique_animation(technique):
         images = list()
         for fn in technique.images:
             image = graphics.load_and_scale(fn)
+            if direction:
```

Comment:
The `PygAnimation` class can handle transformations.  Please use that instead of the low level flip here.  Reason is that I am trying to keep pygame from polluting the entire code base.

should be
```python
tech = PygAnimation(images, False)
tech.flip(True, False)
```
See pyganim.py#455

Reply:
also, remove the pygame import after this is done.